tests/unit-tests/test_embed.py

```diff
@@ -176,6 +179,87 @@ def test_input_layer3(dev):
     th.distributed.destroy_process_group()
     dgl.distributed.kvstore.close_kvstore()
 
+# Test GSgnnEdgeEncoderInputLayer
+@pytest.mark.parametrize("dev", ['cpu','cuda:0'])
+def test_input_layer4(dev):
+    # initialize the torch distributed environment
+    th.distributed.init_process_group(backend='gloo',
+                                      init_method='tcp://127.0.0.1:23456',
+                                      rank=0,
+                                      world_size=1)
+    with tempfile.TemporaryDirectory() as tmpdirname:
+        # get the test dummy distributed graph
+        g, _ = generate_dummy_dist_graph(tmpdirname)
+
+    # Test 1: one edge type and one layer input edge features
+    edge_feat_size = get_edge_feat_size(g, {('n0', 'r0', 'n1'): ['feat']})
+    edge_input_layer = GSEdgeEncoderInputLayer(g, edge_feat_size, 2)
```

Comment:
Can you also test the case when there is no parameters.

Reply:
Added a new test case to check no parameter model.